force-app/main/default/classes/FilesUploadService.cls

```diff
@@ -0,0 +1,58 @@
+/*
+ * @author Lavanya Bonthala
+ * @date   13/09/2022
+ * @description Service Class containing methods specific to FilesUploadService
+ */
+public with sharing class FilesUploadService {
+     /*******************************************************************************************************
+      * @description This method upload files specific to object record
+      * @param RecordId and filedata
+      * @return show success message on file upload
+      */
+    @AuraEnabled
+     public static String uploadFiles(String ParentRecId, String filedata) {
```

Comment:
Change String filedata to List<FileDataWP >. Make this as an instance method. Have AuraEnabled method in the controller of the component and call this service from there